core-bundle/src/Security/Logout/LogoutSuccessHandler.php

```diff
@@ -45,6 +45,12 @@ public function onLogoutSuccess(Request $request): Response
             return $this->httpUtils->createRedirectResponse($request, $targetUrl);
         }
 
+        $targetUrl = $request->request->get('_target_path');
+
+        if (null !== $targetUrl && '' !== $targetUrl) {
```

Comment:
Looks like this could be simplified the same code as on line 44? Also, I would respect the POST parameter before a query, wdyt?

Reply:
was not sure about the precedence here. 🤷‍♂️

Reply:
and the use of `_target_path` only makes sense when it's not empty string, doesn't it?

Reply:
Using `if ($targetUrl = $request->request->get('_target_path')) {` would also skip the empty string so I think we can use the same coding style here.

Reply:
Adjusted in f88532321.